Tasks/AzureFileCopyV2/Utility.ps1

```diff
@@ -247,16 +247,34 @@ function Upload-FilesToAzureContainer
     {
         Write-Output (Get-VstsLocString -Key "AFC_UploadFilesStorageAccount" -ArgumentList $sourcePath, $storageAccountName, $containerName, $blobPrefix)
 
+        $resolvedSourcePath = $sourcePath
+
+        # Check if source path is a file. If yes, then add /Pattern additional argument.
+        if(Test-Path -Path $sourcePath -PathType Leaf)
+        {
+            $fileInfo = Get-Item $sourcePath
+            $resolvedSourcePath = $fileInfo.Directory.FullName
+            $additionalArguments += " /Pattern:$($fileInfo.Name)"
```

Comment:
fileinfo can have white spaces in name here. It should be enclosed in quotes

Reply:
fixed